pkg/reconciler/githubsource/webhook_client.go

```diff
@@ -74,6 +90,10 @@ func (client gitHubWebhookClient) Delete(ctx context.Context, options *webhookOp
 	}
 
 	ghClient := client.createGitHubClient(ctx, options)
+	if alternateGitHubAPIURL != "" {
+		ghClient.BaseURL, _ = url.Parse(alternateGitHubAPIURL)
```

Comment:
Handle the error (might be as simple as returning it).

Reply:
Good spot (pesky underscores) - error now returned